spec/amp-actions-and-events.md

```diff
@@ -72,33 +72,38 @@ For example, the following is possible in AMP.
 ```
 
 ## Element Specific Events
+### amp-carousel
```

Comment:
@choumx I moved things around here a bit. Please take a quick look to make sure I did not mess up your recent changes to this file.